src/Reflectivity-Tools/SemanticVariableIconStyler.class.st

```diff
@@ -0,0 +1,49 @@
+Class {
+	#name : #SemanticVariableIconStyler,
+	#superclass : #IconStylerRealTime,
+	#category : #'Reflectivity-Tools-RealTimeStyling'
+}
+
+{ #category : #styling }
+SemanticVariableIconStyler >> addIconStyle: aNode from: start to: stop color: aColor [
```

Comment:
It seems this method is duplicated in all classes. Maybe I do not see the difference. But it is also a sign to refactor common parts.